src/voxel.h

```diff
@@ -569,6 +580,8 @@ class VoxelManipulator /*: public NodeContainer*/
 	*/
 	u8 *m_flags;
 
+	static const MapNode ContentIgnoreNode;
```

Comment:
But it results in slower code (means I can't return by ref).
What settings did you mean?